ui/src/types/testCases.ts

```diff
@@ -2,3 +2,9 @@ export interface TestCase {
   test: any; // Test from @stellar/anchor-tests
   result?: any; // Result from @stellar/anchor-tests
 }
+
+export type GroupedTestCases = {
+  progress: { running: number; total: number };
+  sep: number;
+  tests: TestCase[];
+}[];
```

Comment:
The figma wireframes also appear to track number of failed tests, do we want to do that here?

Reply:
Good catch. I'll add that here in a future iteration